### PR TITLE
Feature/user registration

### PR DESCRIPTION
**Pull request by Nainterceptor**

### CODE REVIEW

File: pages/edit-account.vue [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+        </div>
+        <div class="column">
+          <ec-field
+            id="phone"
+            label="Code de confirmation courriel"
+          >
+            <div class="field has-addons">
+              <div class="control is-expanded">
+                <input
+                  id="phone"
```
Comment (Rodrive) [inline pages/edit-account.vue]: Should be token here or something like that, not phone. l138 there is already a phone id.

File: pages/edit-account.vue [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+                  v-model="fields.email_token"
+                  v-validate="'required'"
+                  type="text"
+                  class="input"
+                  name="phone"
+                  data-vv-as="Téléphone"
```
Comment (Rodrive) [inline pages/edit-account.vue]: Same here this seems to be wrong

File: pages/edit-account.vue [hunk 3]
```diff
@@ -0,0 +1,339 @@
...
+        </div>
+        <div class="column">
+          <ec-field
+            id="email"
+            label="Code de confirmation téléphone"
+          >
+            <div class="field has-addons">
+              <div class="control is-expanded">
+                <input
+                  id="email"
```
Comment (Rodrive) [inline pages/edit-account.vue]: Same here this seems to be wrong. There is already an 'email' id and we are talking about phone token

File: pages/edit-account.vue [hunk 4]
```diff
@@ -0,0 +1,339 @@
...
+                  v-model="fields.phone.token"
+                  v-validate="'required'"
+                  type="text"
+                  class="input"
+                  name="email"
+                  data-vv-as="Email"
```
Comment (Rodrive) [inline pages/edit-account.vue]: Same here

File: pages/edit-account.vue [hunk 5]
```diff
@@ -0,0 +1,339 @@
...
+          >
+            <div class="field has-addons">
+              <div class="control is-expanded">
+                <input
+                  id="email"
+                  v-model="fields.phone.token"
+                  v-validate="'required'"
+                  type="text"
+                  class="input"
+                  name="email"
```
Comment (Rodrive) [inline pages/edit-account.vue]: Same here

File: pages/login.vue [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+        field-id="email"
+        :icon-left="['fas', 'user']"
+      >
+        <input
+          id="email"
+          v-model="user.email"
+          required="required"
+          class="input is-white"
+          type="email"
+          placeholder="john@doe.com"
```
Comment (Rodrive) [inline pages/login.vue]: I think I saw a placeholder with "john@doe.tld" previously. For consistency, maybe we should have the same placeholder everywhere.

Comment (Nainterceptor) [reply pages/login.vue]: Done

File: pages/edit-account.vue (hunk 5 above)
Comment (Nainterceptor) [reply pages/edit-account.vue]: Done

File: pages/edit-account.vue (hunk 1 above)
Comment (Nainterceptor) [reply pages/edit-account.vue]: Done

File: pages/edit-account.vue (hunk 2 above)
Comment (Nainterceptor) [reply pages/edit-account.vue]: Done

File: pages/edit-account.vue (hunk 3 above)
Comment (Nainterceptor) [reply pages/edit-account.vue]: Done

File: pages/edit-account.vue (hunk 4 above)
Comment (Nainterceptor) [reply pages/edit-account.vue]: Done

**Pull request merged**